flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/FileSystemITCaseBase.scala

```diff
@@ -368,6 +372,63 @@ trait FileSystemITCaseBase {
     )
   }
 
+  @Test
+  def testReadAllMetadata(): Unit = {
```

Comment:
Why are these tests in scala? 🙈

Reply:
We have all our test infra in scala, and this particular class is then used to run the same test cases for every format, that's why i need this test here :)